docs/storage/prometheus.md

```diff
@@ -31,6 +31,8 @@ Metric name | Type | Description | Unit (where applicable) | -disable_metrics pa
 `container_cpu_system_seconds_total` | Counter | Cumulative system cpu time consumed | seconds | |
 `container_cpu_usage_seconds_total` | Counter | Cumulative cpu time consumed | seconds | |
 `container_cpu_user_seconds_total` | Counter | Cumulative user cpu time consumed | seconds | |
+`container_cpu_userspace_usage_seconds` | Counter | Time spent in userspace per CPU per container | seconds | |
```

Comment:
Maybe shorter names of metrics will be better?
I think about:
- container_cpu_user_seconds
- container_cpu_kernel_seconds

Reply:
Did that, but now you get container_cpu_user_seconds and container_cpu_user_seconds_total in one output. In general i think it is ok, but I'm bringing it up for general awareness.

Reply:
I've seen container_cpu_user_seconds_total metric, imho container_cpu_user_seconds is the proper name of metric but we will see what maintainer decides.